tools/postgres/schema.sql

```diff
@@ -98,6 +98,7 @@ CREATE TABLE webknossos.dataSets(
   logoUrl VARCHAR(2048),
   sortingKey TIMESTAMPTZ NOT NULL,
   details JSONB,
+  tags VARCHAR(256)[] NOT NULL DEFAULT '{}',
```

Comment:
We could also use the `json` or `jsonb` type (seems like it's available from pg 9.6). It's also possible to create indices on jsonb columns and query them efficiently (e.g. when searching/selecting by tag):

https://www.postgresql.org/docs/9.4/datatype-json.html#JSON-INDEXING

Reply:
But also totally fine as-is for now I guess.